lib/components/result-view/transforms.js

```diff
@@ -1,26 +1,56 @@
 /* @flow */
 
-import {
-  standardTransforms,
-  standardDisplayOrder,
-  registerTransform
-} from "@nteract/transforms";
+import React from "react";
+import { Media } from "@nteract/outputs";
 import PlotlyTransform from "@nteract/transform-plotly";
 import { VegaLite1, VegaLite2, Vega2, Vega3 } from "@nteract/transform-vega";
 
+PlotlyTransform.defaultProps = {
+  ...PlotlyTransform.defaultProps,
+  mediaType: PlotlyTransform.MIMETYPE
```

Comment:
If `mediaType` is not already a default prop on these transforms, I wonder if it should be. Might be a quick PR to the monorepo.

Reply:
Yeah wouldn't be hard to do. I just don't know how to propose things to them on the mono @BenRussert